src/components/SecondaryPanes/Frames/tests/FrameMenu.js

```diff
@@ -0,0 +1,73 @@
+const originalL10N = L10N;
+import FrameMenu from "../FrameMenu";
+import { showMenu } from "devtools-launchpad";
+import { copyToTheClipboard } from "../../../../utils/clipboard";
+jest.mock("devtools-launchpad", () => ({ showMenu: jest.fn() }));
+jest.mock("../../../../utils/clipboard", () => ({
+  copyToTheClipboard: jest.fn()
+}));
+
+function generateMockCopySource(label, accesskey) {
```

Comment:
so! I think i found a nice compromise -- I added a `ContextMenuItem` type, and am now just testing for the ids. I also rewrote the functions a bit so that testing the id's should be sufficient for the formatting